### DISCUSSION TITLE
is LDPC going to be supported in future?

### DISCUSSION THREAD

**Discussion by rookiebeotch** [open-sdr/openwifi#430] [General]
is LDPC FEC encoding/decoding going to be supported in future?

**Comment by tryss** [comment]
I'm new to that project but i think LDPC is already running in OFDM TX as FEC encoder. tell me if i'm wrong ?

**Comment by rookiebeotch** [comment]
I don't think so. it does convolutional encoding and decoding.  When tested
with LDPC transmitter this receiver rejects it as unsupported type.

**Comment by tryss** [comment]
Ok so i will try to implement my LDPC code, i will let you know if it works with the Viterbi.

**Comment by robgar2001** [comment]
Currently, we do not support LDPC. It is a requirement if we want to go to higher MCS. Doing so is not our current priority.

**Comment by moh-amra** [comment]
> is LDPC FEC encoding/decoding going to be supported in future?

I think there will be much work to do to support the other standards.

**Reply by robgar2001** [reply]
Given enough tokens and a good test bench, I am convinced that AI would likely succeed in implementing LDPC.
